rainier-core/src/main/scala/com/stripe/rainier/core/Categorical.scala

```diff
@@ -41,9 +41,6 @@ final case class Categorical[T](pmf: Map[T, Real]) extends Distribution[T] {
     }
   }
 
-  def toMixture[V](implicit ev: T <:< Distribution[V]): Mixture[V, T] =
```

Comment:
You can still have this but where `ev: T <:< Continuous`

Reply:
Kk, done